### PR TITLE
update translation file

### PR DESCRIPTION
**Pull request by ocke**
\cc @zendesk/vegemite @zendesk/i18n 

Update translation file

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -40,45 +40,56 @@ By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://w
...
-      screenshot: "https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo"
+      title: "iframe URL"
+      screenshot: "https://drive.google.com/open?id=17ADg5SU3yfTPzv0s8SSqFzml3fb40fVy"
       value: "iframe URL"
   - translation:
       key: "txt.apps.iframe_app.app.parameters.inactiveIconURL.label"
       title: "Inactive Icon URL. See https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo"
       screenshot: "https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo"
       value: "Inactive icon URL"
+      obsolete: 18-5-2018
```
Comment (hiroiguchi) [inline translations/en.yml]: Obsolete date format is yyyy-mm-dd, so can you update this to following?

`obsolete: "2018-05-18"`

File: translations/en.yml [hunk 2]
```diff
@@ -40,45 +40,56 @@ By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://w
...
   - translation:
       key: "txt.apps.iframe_app.app.parameters.hoverIconURL.label"
       title: "Hover Icon URL. See https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo"
       screenshot: "https://zendesk.box.com/s/bys22b6tldnopxuw0g3qhe3nhoaseqqo"
       value: "Hover icon URL"
+      obsolete: 18-5-2018
```
Comment (hiroiguchi) [inline translations/en.yml]: Obsolete date format is yyyy-mm-dd, so can you update this to following?

`obsolete: "2018-05-18"`

File: translations/en.yml [hunk 3]
```diff
@@ -40,45 +40,56 @@ By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://w
...
-      value: "Check this box if the iframed website requires ZAF SDK support. See the Iframes in Apps topic in our developer documentation."
+      title: "Help text to explain the ZAF SDK support option."
+      screenshot: "https://drive.google.com/open?id=17ADg5SU3yfTPzv0s8SSqFzml3fb40fVy"
+      value: "Check this box if the iframed website requires ZAF SDK support. See [Iframes in Apps](https://developer.zendesk.com/apps/docs/developer-guide/using_sdk)."
   - translation:
       key: "txt.apps.iframe_app.app.description"
       title: "Embed any website in your Zendesk (formerly known as Sidebar Icon app). See https://zendesk.box.com/s/0c5a122j1hdgbx4yu6bqlktdtl1qfu03"
       screenshot: "https://zendesk.box.com/s/0c5a122j1hdgbx4yu6bqlktdtl1qfu03"
       value: "Embed any website in your Zendesk (formerly known as Sidebar Icon app)"
+      obsolete: 18-5-2018
```
Comment (hiroiguchi) [inline translations/en.yml]: Obsolete date format is yyyy-mm-dd, so can you update this to following?

`obsolete: "2018-05-18"`

File: translations/en.yml [hunk 4]
```diff
@@ -40,45 +40,56 @@ By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://w
...
+  - translation:
+      key: "txt.apps.iframe_app.app.parameters.icon.label"
+      title: "Selected Icon"
+      screenshot: "https://drive.google.com/open?id=17ADg5SU3yfTPzv0s8SSqFzml3fb40fVy"
+      value: "Icon characters or image name"
+  - translation:
+      key: "txt.apps.iframe_app.app.parameters.icon.helpText"
+      title: "Icon help text"
+      screenshot: "https://drive.google.com/open?id=17ADg5SU3yfTPzv0s8SSqFzml3fb40fVy"
+      value: "The Iframe Icon will be the the first two (UTF8) characters provided (e.g. `WI` or `øå` or `✈☃`), [UTF8 Icons](https://www.utf8icons.com/). Alternatively, if an id is provided surrounded by colons `:`, it will use an icon from either [Icon Set 1](https://garden.zendesk.com/svg-icons/16px/) or [Icon Set 2](https://garden.zendesk.com/svg-icons/26px/) (e.g. `:zd-svg-icon-16-table-fill:` or `:zd-svg-icon-26-home-fill:`)."
```
Comment (hiroiguchi) [inline translations/en.yml]: I think `The Iframe Icon` should be `The Iframe icon`. Can you check if the `Icon` is in lower case?
If this has to be in lower case, please also update screenshot.

**Comment by ocke** [comment]
@hiroiguchi updated :+1:

File: translations/en.yml [hunk 5]
```diff
@@ -40,45 +40,55 @@ By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://w
...
+      obsolete: "2018-05-18"
+  - translation:
+      key: "txt.apps.iframe_app.app.parameters.icon.label"
+      title: "Selected Icon"
+      screenshot: "https://drive.google.com/open?id=1G9VkNWqC4cgea8Yxy1h-d1QQI6W9sxMZ"
+      value: "Icon characters or image name"
+  - translation:
+      key: "txt.apps.iframe_app.app.parameters.icon.helpText"
+      title: "Icon help text"
+      screenshot: "https://drive.google.com/open?id=1G9VkNWqC4cgea8Yxy1h-d1QQI6W9sxMZ"
```
Comment (hiroiguchi) [inline translations/en.yml]: Thank you for updating the value but can you also update screenshot please? (It's `The Iframe Icon` in the screenshot)

**Comment by hiroiguchi** [comment]
:ok for string. Thank you @ocke

**Pull request merged**